index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/f9ba107a19.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <main>
+      <section class="bg">
+        <header class="header-align">
+          <div class="logo"><img src="./assets/images/logo.png" alt="" /></div>
+          <nav>
+            <a href="#">
+              <img src="./assets/images/union.png" alt="" />
+            </a>
+          </nav>
+        </header>
+
+        <section class="Headline">
+          <h1>I'm Nohain. <br />Glad to see you!</h1>
+          <p class="myself">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hesitate to
+            contact me.
+          </p>
+          <p class="connect-p">LET’S CONNECT</p>
+          <ul>
+            <li>
+              <a href="https://twitter.com/NohainZ"
+                ><i class="fa-brands fa-twitter"></i
+              ></a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/in/nohain-islam-0261b4148/"
+                ><i class="fa-brands fa-linkedin-in"></i
+              ></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-medium"></i></a>
+            </li>
+            <!-- Attn: Dear Reviewr; This logo given in figma 
+                is a very old logo of https://medium.com/ I cannot find the old 
+                logo anywhere so I had to import the new Medium logo here; -->
+            <li>
+              <a href="https://github.com/Zafron047"
+                ><i class="fa-brands fa-github"></i
+              ></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-angellist"></i></a>
+            </li>
```

Comment:
- [ ] [OPTIONAL] For good user experience, also to allow others (potential recruiters) to connect with you, it would be nice if you can add your profile links in the `href` attributes of these links so that when clicked, you can be contacted for any purpose. This improves accessibilty and user experience :+1:

Reply:
@AbayomiOlaoye Noted. I will create accounts for those missing `li` `href `within few days and will sync in the link in there very soon.